.ci.yaml

```diff
@@ -322,8 +322,6 @@ targets:
         [
           {"dependency": "goldctl", "version": "git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603"}
         ]
-      os: Mac-10.15
-      xcode: 12c33 # xcode 12.3
```

Comment:
Are you trying to validate logics without xcode?
(I created https://github.com/flutter/engine/pull/34051 to bump the xcode to 13 (same as this PR as of now))

If yes, then this PR is not enough as the platform values will be populated to all Mac targets. To do that, we should remove platform values, and then add xcode values to each Mac target excpet `Mac Web Engine`.

If no, feel free to close [mine](https://github.com/flutter/engine/pull/34051) or dup this one.

Reply:
Due to the inherited properties of the mac target, I actually wasn't sure whether this change would prevent Xcode from being installed or not. It appears we do install Xcode, which is fine because that's the behavior we want for now. We can twiddle with the Mac targets later if we want to try to prevent Xcode from being installed in the web_engine tests.

You were quicker to get your PR in, so let's just merge yours and close this one.